paf_utilities/README

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
Assuming this is a placeholder for the documentation :)

Reply:
Yep!